### PR TITLE
refine autograd docs

### PR DESCRIPTION
**Pull request by larroy**

### CODE REVIEW

**Comment by larroy** [comment]
@mli is there a way to see the book output without having to compile locally and execute all the notebooks? is very slow in my machine.

**Comment by mli** [comment]
You can click the "All notebook" link on the navbar, or here is the link: https://en.d2l.ai/d2l-en.zip

**Comment by larroy** [comment]
I meant from a PR.

File: chapter_crashcourse/autograd.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ with autograd.record():
 print(y)
 ```
 
-Since the shape of `x` is (4, 1), `y` is a scalar. Next, we can automatically find the gradient by calling the `backward` function. It should be noted that if `y` is not a scalar, MXNet will first sum the elements in `y` to get the new variable by default, and then find the gradient of the variable with respect to `x`.
+Since the shape of `x` is (4, 1), `y` is a scalar. Next, we can automatically find the gradient of all the inputs by calling the `backward` function. It should be noted that if `y` is not a scalar, MXNet will first sum the elements in `y` to get the new variable by default, and then find the analytical gradient of the variable with respect to `x` evaluated at `x` that is $\frac{dy}{dx}\bigr\rvert_{x=x_0}$.
```
Comment (astonzhang) [inline chapter_crashcourse/autograd.md]: x_0 is undefied

**Comment by larroy** [comment]
When I execute d2lbook it executes the notebooks, is it possible to skip them? in CI seems you go much faster.

**Comment by mli** [comment]
You can change `eval_notebook = True` to False in config.ini to disable notebook execution. I'll add "push PR build results to a staging repo" as a TODO

**Pull request merged**